### PR TITLE
T4 : LCM and GCD

### PR DESCRIPTION
**Pull request by RaaneshKumar**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Find LCM and GCD of two numbers
+
+Console.Write ("Enter the first number: ");
```
Comment (rajeevramanTMM) [inline Program.cs:3]: Fix these errors.
image
image

File: Program.cs [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+string first_number = Console.ReadLine ();
+Console.Write ("Enter the second number: ");
+string second_number = Console.ReadLine ();
+int number1 = int.Parse (first_number);
+int number2 = int.Parse (second_number);
+int[] multiples = new int[] { };
```
Comment (rajeevramanTMM) [inline Program.cs]: Use a variable to store the maximum value no need an array for getting the maximum value.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
-    }
-}
\ No newline at end of file
+﻿// Find LCM and GCD of two numbers
+
+Console.Write ("Enter the first number: ");
+bool num1 = int.TryParse (Console.ReadLine (), out int number1);
+Console.Write ("Enter the second number: ");
+bool num2 = int.TryParse (Console.ReadLine (), out int number2);
+if (num1 == true && num2 == true) {
```
Comment (rajeevramanTMM) [inline Program.cs]: No need to explicitly check a bool with true. if (num1 && num2) is enough.

File: Program.cs [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
+   int gcd = 0;
+   if (number1 == 0) Console.WriteLine ($"LCM of given numbers is 0. \nGCD of given numbers is {number2}.");
```
Comment (rajeevramanTMM) [inline Program.cs]: Here you are checking number1 with zero. Is this is expected output which I have shared in the image?
image

File: Program.cs [hunk 5]
```diff
@@ -1,10 +1,18 @@
...
+   if (number1 > 0) {
+      for (int i = 1; i <= number1; i++) {
+         int lcm1 = number1 % i; int lcm2 = number2 % i;
```
Comment (rajeevramanTMM) [inline Program.cs]: No need lcm1 and lcm2 you can compare the operation inside if statement itself. It has only one instance of it.

File: Program.cs [hunk 6]
```diff
@@ -1,10 +1,18 @@
...
+         if (lcm1 == 0 && lcm2 == 0 && i > gcd) gcd = i;
+      }
+      int lcm = (number1 * number2) / gcd;
```
Comment (rajeevramanTMM) [inline Program.cs]: lcm is one instance can you change?.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 4 above)
Comment (RaaneshKumar) [reply Program.cs]: Yes, this is the expected output.

**Pull request closed without merging**